utils/settings/character.py

```diff
@@ -6,6 +6,7 @@
 
 from utils.functions import get_positivity
 from . import SettingsBaseModel
+from ..enums import CoinsAutoConvert
```

Comment:
nit: `from ..enums` -> `from utils.enums`; both are valid but the absolute import is preferred here

Reply:
Is there a way to set pycharm to do this automatically? I used the context manager to add the import

Reply:
<img width="521" alt="Screenshot 2022-03-29 at 10 00 42" src="https://user-images.githubusercontent.com/8866981/160666104-225558dd-28f1-4921-b1ad-3deb3ac1581f.png">
There's this but I don't think there's any more fine grained selection for absolute vs relative imports :(

Reply:
Gotcha, I'll keep an eye out for this in the future regardless!